### PR TITLE
feat: added products page

### PR DESCRIPTION
**Pull request by arjundevn**
productsPage

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -68,7 +62,7 @@
     border-radius: 10%;
 }
 
-.categories img:hover {
+.categories img:hover, .card:hover {
     box-shadow: rgba(256, 256, 256, 0.35) 10px 10px 15px;
```
Comment (sushil-mod) [inline styles/index.css:66]: use color variable

File: styles/products.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.filter-brand{
+    display:block;
+}
+
+.products-listings{
+    padding: -1rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (sushil-mod) [inline styles/products.css]: its right , you can also try repeat(3,fr) in grid-template-columns (tell me if i wrong )

**Review by sushil-mod — APPROVED** [review]
Added few comment, rest is good

File: styles/index.css (hunk 1 above)
Comment (arjundevn) [reply styles/index.css:66]: Color variable does not work inside box property: https://stackoverflow.com/questions/59250927/box-shadow-with-rgba-variable

File: styles/products.css (hunk 2 above)
Comment (ajit-sharmaGH) [reply styles/products.css]: repeat(3, 1fr) ;

**Review by ajit-sharmaGH — APPROVED** [review]
Looks good to me

File: styles/products.css (hunk 2 above)
Comment (arjundevn) [reply styles/products.css]: Yes, you are right 👍🏽 repeat(3, 1fr)

**Pull request merged**